### ISSUE TITLE
Death screen appears only after the player rejoins with Release2Beta

### ISSUE THREAD

**Issue by E404NNF** [GeyserMC/Geyser#1394]
**Describe the bug**

Death screen appears only after the player rejoins with Release2Beta
**To Reproduce**

1. Join a beta server through R2B
2. Die using anyway (i tested only with /kill)
3. Observe.

**Expected behavior**

Death screen to appear without rejoining

**Screenshots / Videos**

**Server Version**

CraftBukkit 1060-1090 (and it's forks)
Release2Beta Latest Commit

**Geyser Version**

**Minecraft: Bedrock Edition Version**

Build 420
**Additional Context**

`Plugins (21/21): AuthMe, BetaItemStacker, BlocksOnGlass, DiscordChatBridge, DiscordCore, DynamicLight, Essentials, EssentialsChat, EssentialsSpawn, EvolutionCore, HeroicDeath, LWC, NoCheat, PermissionsEx, ReleaseToBeta, SleepPercent, StairsDrop, TPSPlugin, WorldBorder, WorldEdit, WorldGuard`

https://dump.geysermc.org/R50t7Bp5OehOnDpU5TgnA7aniCYPn9Mn

**Comment by E404NNF** [comment]
This was fixed by build 425 somehow...

**Comment by E404NNF** [comment]
Nvm it didn't

**Comment by Jens-Co** [comment]
Hello @E404NNF are you still having problems with the deathscreen ? If not can you close this issue ? Since this happend on an beta release I assume this can be closed. If not responding I'll close this issue within a few days.
Jens

**Comment by Chew** [comment]
If you are able to reproduce this, please open a new issue.
